cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server_data_adapters.go

```diff
@@ -25,15 +25,9 @@ import (
 	log "k8s.io/klog/v2"
 )
 
-func (s *Server) buildAvailablePackageSummary(pkgMetadata *datapackagingv1alpha1.PackageMetadata, pkgVersionsMap map[string][]pkgSemver, cluster string) (*corev1.AvailablePackageSummary, error) {
+func (s *Server) buildAvailablePackageSummary(pkgMetadata *datapackagingv1alpha1.PackageMetadata, latestVersion string, cluster string) *corev1.AvailablePackageSummary {
```

Comment:
There was no requirement for having the package map, just the latest version, which also means we don't need the ability to return an error here.